### PR TITLE
Build and archive all JITs in the JIT rolling build

### PR DESCRIPTION
**Pull request by BruceForstall**

### CODE REVIEW

**Comment by BruceForstall** [comment]
This does increase the size of the upload from about 19MB to about 907MB, mostly because of the debug info. We could re-evaluate this (either compress the debug info or don't include it in the uploads) if it becomes a problem.

**Comment by BruceForstall** [comment]
@AndyAyersMS @kunalspathak @dotnet/jit-contrib PTAL

**Comment by BruceForstall** [comment]
Given the significant size increase of the build upload including the debug info, I've changed the code to not upload the debug info for now. If we find there is a real need for the debug info later, we can change this.

File: src/coreclr/scripts/jitrollingbuild.py [hunk 1]
```diff
@@ -262,6 +272,42 @@ def upload_blob(file, blob_name):
...
+    # so we don't want to pick that up.
+
+    if coreclr_args.host_os == "OSX":
+        allowed_extensions = [ ".dylib" ]
+        # Add .dwarf for debug info
+    elif coreclr_args.host_os == "Linux":
+        allowed_extensions = [ ".so" ]
+        # Add .dbg for debug info
+    elif coreclr_args.host_os == "Windows_NT":
+        allowed_extensions = [ ".dll" ]
```
Comment (kunalspathak) [inline src/coreclr/scripts/jitrollingbuild.py:295]: curious, why is `allowed_extensions` an array?

**Review by kunalspathak — APPROVED** [review]
LGTM with 1 question.

File: src/coreclr/scripts/jitrollingbuild.py (hunk 1 above)
Comment (BruceForstall) [reply src/coreclr/scripts/jitrollingbuild.py:295]: For future flexibility, such as adding debug info. Currently it doesn't need to be.

**Pull request merged**